docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -99,7 +99,11 @@ The alert summary section provides general alert details such as the alert's sta
 
 The alert's reason statement is under the alert summary. The reason statement describes what generated the alert and provides general alert details. You can use this information to understand the alert's origin and determine if the alert is relevant to your investigation.
 
-The *Highlighted Fields* section is also provided for all alerts and is customized to only show the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.
+The alert details flyout also lists the number and names of cases to which the alert has been added. Click case names to open case details.
```

Comment:
```suggestion
The alert details flyout also lists the number and names of cases to which the alert has been added. Click a case's name to view its details.
```
Could be wrong but if I understand correctly you would click on one case name.

Reply:
You're correct and this makes sense - thanks @benironside ! I do want to stick with "open" over "view" though to avoid using (what could be construed as) ableist language.